public/contribute/_overview_zh.md

```diff
@@ -0,0 +1,23 @@
+# 对Crosswalk的贡献
+
+Crosswalk是一款开源项目。源代码存放在[GitHub](http://github.com/crosswalk-project)上。Bugs, features和issues都通过[JIRA](https://crosswalk-project.org/jira)处理。
```

Comment:
the github address seems to be a folder of repoes. is it proper to point to this place? @lisa0314 @jondong @chuan9